### PR TITLE
docs: fix protect all url with `$globals`

### PR DESCRIPTION
**Pull request by datamweb**
ref: https://github.com/codeigniter4/shield/discussions/441#discussioncomment-3672007
It seems that in https://github.com/codeigniter4/shield/pull/429#issuecomment-1242676793 I meant to _rewrite_ filter `session`, but I didn't, that was my carelessness, sorry.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -149,8 +149,18 @@ These can be used in any of the [normal filter config settings](https://codeigni
...
+        // 'honeypot',
+        // 'csrf',
+        // 'invalidchars',
+        'session' => ['except' => ['login*', 'register*']],
+    ],
+    'after' => [
+        'toolbar',
+        // 'honeypot',
+        // 'secureheaders',
+    ],
```
Comment (kenjis) [inline docs/install.md:163]:
```suggestion
    'before' => [
        // ...
        'session' => ['except' => ['login*', 'register*']],
    ],
    // ...
```

File: docs/install.md [hunk 2]
```diff
@@ -149,8 +149,18 @@ These can be used in any of the [normal filter config settings](https://codeigni
 If you want to limit all routes (e.g. `localhost:8080/admin`, `localhost:8080/panel` and ...), you need to add the following code in the `app\Config\Filters.php` file.
```
Comment (kenjis) [inline docs/install.md:149]:
```suggestion
If you want to limit all routes (e.g. `localhost:8080/admin`, `localhost:8080/panel` and ...), you need to add the following code in the `app/Config/Filters.php` file.
```

**Review by kenjis — APPROVED** [review]
LGTM!

**Review by MGatner — APPROVED** [review]
Thanks for the quick fix!

**Pull request merged**